tests.py

```diff
@@ -116,318 +139,494 @@ def test_version_number_in_setup(self):
-        api.client = HttpClientMock({'users': (200,
-            json.dumps([{'name': 'first user'}, {'name': 'second user'}]))})
-
-        def fn():
-            api.set_toopher_enabled_for_user('multiple users', True)
-        self.assertRaises(toopher.ToopherApiError, fn)
-
-    def test_enable_toopher_no_users(self):
-        api = toopher.ToopherApi('key', 'secret')
-        api.client = HttpClientMock({'users': (200, '[]')})
-
-        def fn():
-            api.set_toopher_enabled_for_user('no users', True)
-        self.assertRaises(toopher.ToopherApiError, fn)
 
+class ZeroStorageTests(unittest.TestCase):
+    def setUp(self):
+        self.api = toopher.ToopherApi('key', 'secret')
+        self.id = str(uuid.uuid4())
+        self.terminal_name = 'terminal_name'
+        self.requester_terminal_id = 'requester_terminal_id'
+        self.user = {
+            'id': str(uuid.uuid4()),
+            'name': 'user_name',
+            'disable_toopher_auth': False
+        }
+        self.user_id = self.user['id']
+        self.user_name = self.user['name']
 
+    def test_create_user_terminal(self):
+        self.api.advanced.raw.client = HttpClientMock({
+            'user_terminals/create': (200,
+                json.dumps({
+                    'id': self.id,
+                    'name': self.terminal_name,
+                    'name_extra': self.requester_terminal_id,
+                    'user': self.user
+                })
+            )
+        })
+        user_terminal = self.api.advanced.user_terminals.create(self.user_name, self.terminal_name, self.requester_terminal_id)
+        self.assertEqual(self.api.advanced.raw.client.last_called_method, 'POST')
+        self.assertEqual(user_terminal.id, self.id)
+        self.assertEqual(user_terminal.user.name, self.user_name)
+        self.assertEqual(user_terminal.user.id, self.user_id)
+        self.assertEqual(user_terminal.name, self.terminal_name)
+        self.assertEqual(user_terminal.name_extra, self.requester_terminal_id)
+
+    def test_get_user_terminal_by_id(self):
+        self.api.advanced.raw.client = HttpClientMock({
+            'user_terminals/{0}'.format(self.id): (200,
+                json.dumps({
+                    'id': self.id,
+                    'name': self.terminal_name,
+                    'name_extra': self.requester_terminal_id,
+                    'user': self.user
+                })
+            )
+        })
+        user_terminal = self.api.advanced.user_terminals.get_by_id(self.id)
+        self.assertEqual(self.api.advanced.raw.client.last_called_method, "GET")
+        self.assertEqual(user_terminal.id, self.id)
+        self.assertEqual(user_terminal.name, self.terminal_name)
+        self.assertEqual(user_terminal.name_extra, self.requester_terminal_id)
+        self.assertEqual(user_terminal.user.name, self.user_name)
+        self.assertEqual(user_terminal.user.id, self.user_id)
+
+    def test_get(self):
```

Comment:
This is testing the raw get method from the ApiRawRequester. I've changed the name to test_raw_get. Should it be more descriptive?